File: python/problem1.py
# Project Euler problem 1: https://projecteuler.net/problem=1
# Author: Tom Harkness
# May 29, 2021

if __name__ == "__main__":
    print(sum([_ for _ in range(1,1000) if _ % 3 == 0 or _ % 5 == 0]))

File: c/problem1.c
// Project Euler problem 1: https://projecteuler.net/problem=1
// Author: Tom Harkness
// May 27, 2021

#include <stdio.h>

int main(void) {
    size_t result = 0;
    
    for (size_t i = 0; i < 1000; i++) {
        if (i % 3 == 0 || i % 5 == 0) {
            result += i;
        }
    }

    printf("answer: %zu", result);
    return 0;
}
    